toes/triggers.py

```diff
@@ -134,6 +141,26 @@ def trigger_reaction_custom(*, keyword: str, probability: float = 1.0, emoji_id:
     async def trigger(bot: Client, message: Message): ...
     return trigger
 
+@jason_trigger('reaction_trail_standard')
+def trigger_reaction_standard(*, author_id: str, probability: float = 1.0, emoji: str,  **kwargs: Any) -> Trigger:
```

Comment:
We've got a naming collision with the new factory methods you added. Could you change the names to differentiate them?

Reply:
fixed